### ISSUE TITLE
Vessel is added to build list although build constraints are not fulfilled

### ISSUE THREAD

**Issue by malnvenshorn** [magico13/KCT#181]
I'm new to this mod and I noticed that vessels are added to the build list although build constraints are not fulfilled, e.g. if part limit is exceeded.

**Comment by magico13** [comment]
You typically can build the vessel but cannot launch it until the requirements are met. I don't recall if it warns you at build time that you'll need to upgrade before launch, but it should inform you when you try to launch it.

**Comment by malnvenshorn** [comment]
Yes some warning pops up, but I would like to turn this off. It happened a few times that I forgot to remove the specific build from the queue, so it was blocking other builds.

Besides that. The part limit is attached to the VAB, so if the constraint fails you shouldn't be able to build this. Of course this argument doesn't count for the weight and size constraint, because those refer to the launch pad.
